source/Ship.cpp

```diff
@@ -3355,10 +3361,17 @@ double Ship::HeatDissipation() const
 // Get the maximum heat level, in heat units (not temperature).
 double Ship::MaximumHeat() const
 {
-	return MAXIMUM_TEMPERATURE * (cargo.Used() + attributes.Mass() + attributes.Get("heat capacity"));
+	return attributes.Get("maximum temperature") / 10. * (cargo.Used() + attributes.Mass()
```

Comment:
`"maximum temperature"` is always divided by at least 10, just drop a zero?

Reply:
you could alternatively have the files use 100 by default, and multiply that by 10 to display as 1000 K